core/sms_server.py

```
@@ -0,0 +1,1422 @@
+
+
+@app.post("/test/register_mobile")
+async def test_register_mobile(data: OnboardingRequest):
+    """
+    Register mobile for onboarding via Test UI.
+    Returns hash and message format for validation.
+    """
+    try:
+        # Call the main onboarding endpoint
+        result = await register_mobile_production_2(data)
+        return {"success": True, "data": result}
```

Ruff (0.14.6)

Consider moving this statement to an `else` block

(TRY300)